### PR TITLE
Add Partner Wallets and Modify Wallets Page

### PR DESCRIPTION
**Pull request by DannyS03**
- Wallets page can be the general getting started page for all things wallets (will follow more updates)
- Adds table for partner wallets
- Redirects prev. content to this page

### CODE REVIEW

File: docs/develop/wallets/getting-started.md [hunk 1]
```diff
@@ -9,19 +9,49 @@ keywords:
 image: https://matic.network/banners/matic-network-16x9.png 
 ---
 
-The following article aims to be a guide through integration of a key management strategy on client side of your Decentralised Application on Matic Network.
 
-The following wallets will be discussed:
+This guide includes wallets that support the Polygon network which Polygon Technology is in
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:13]: I think we ought to rephrase this

File: docs/develop/wallets/getting-started.md [hunk 2]
```diff
@@ -9,19 +9,49 @@ keywords:
...
+| [Pillar](https://www.pillar.fi/) 	| non-custodial 	| EOA 	| no 	| interface 	| no 	|
+| [Rainbow](https://rainbow.me/) 	| non-custodial 	| EOA 	| no 	| interface 	| yes 	|
+| [Sequence](https://sequence.app/auth) 	| non-custodial 	| smart contract 	| no 	| interface 	| no 	|
+| [SimpleHold](https://simplehold.io/) 	| non-custodial 	| EOA 	| no 	| no 	| no 	|
+| [SteakWallet](https://steakwallet.fi/) 	| non-custodial 	| EOA 	| no 	| interface 	| no 	|
+| [TokenPocket](https://www.tokenpocket.pro/en) 	| non-custodial 	| EOA 	| no 	| support 	| yes 	|
+| [Torus](https://toruswallet.io/) 	| non-custodial 	| EOA 	| no 	| support 	| no 	|
+| [Unstoppable](https://unstoppable.money/) 	| non-custodial 	| EOA 	| no 	| no 	| yes 	|
+| [Venly](https://www.venly.io/) 	| hybrid 	| smart contract 	| no 	| interface 	| no 	|
+| [XDeFi](https://www.xdefi.io/) 	| non-custodial 	| EOA 	| no 	| interface 	| no 	|
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:52]: there is just one hiccup here: how do we know this is an extensive list that really mentions ALL wallets supporting MATIC

**Review by cerberushades — CHANGES_REQUESTED** [review]

File: docs/develop/wallets/getting-started.md (hunk 2 above)
Comment (DannyS03) [reply docs/develop/wallets/getting-started.md]: the page mentions that these are wallets polygon is in partnership with

Comment (DannyS03) [reply docs/develop/wallets/getting-started.md]: addressed

File: docs/develop/wallets/getting-started.md (hunk 1 above)
Comment (DannyS03) [reply docs/develop/wallets/getting-started.md]: took it out - an update will follow that gives an intro to wallets and flows into this material

**Comment by DannyS03** [comment]
> @cerberushades We need also to add the "Manual" Wallets from here (not only Native one):
> 
> https://www.notion.so/polygontechnology/Wallet-Matrix-8e699080c29f4c248bca75f40e2eb690

The ones that don't support Polygon?

File: docs/develop/wallets/getting-started.md [hunk 3]
```diff
@@ -37,14 +99,13 @@ The following wallets will be discussed:
 
 * [**Wallet Connect**](https://walletconnect.org/): WalletConnect is an open protocol to communicate securely between Wallets and Dapps (Web3 Apps). The protocol establishes a remote connection between two apps and/or devices using a Bridge server to relay payloads. These payloads are symmetrically encrypted through a shared key between the two peers. [For more...](walletconnect)
 
-
 ### High-level steps:
 
 The overall steps would essentially remain the same for any client side application to talk to the blockchain:
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:104]:
```suggestion
The overall steps would essentially remain the same for any client-side application to talk to the blockchain:
```

File: docs/develop/wallets/getting-started.md [hunk 4]
```diff
@@ -37,14 +99,13 @@ The following wallets will be discussed:
...
- 
-1. **Set up Web3**: [web3.js](https://web3js.readthedocs.io/) is a javascript library that allows our client-side application to talk to the blockchain. We configure web3 to communicate via Metamask/Wallet Connect/Portis. 
-> Note: Refer [Web3.js](https://web3js.readthedocs.io/en/v1.2.2/getting-started.html#adding-web3-js) docs to 
-add web3 to your project 
-2. **Set up Account**: To send transactions from (specifically for transactions that alter the state of the blockchain) 
-3. **Instantiate contracts**: Once we have our web3 object in place, we next instantiate our deployed contract, with which we interact. 
+
+1. **Set up Web3**: [web3.js](https://web3js.readthedocs.io/) is a javascript library that allows our client-side application to talk to the blockchain. We configure web3 to communicate via Metamask/Wallet Connect/Portis.
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:106]:
```suggestion
1. **Set up Web3**: [web3.js](https://web3js.readthedocs.io/) is a javascript library that allows our client-side application to talk to the blockchain. We configure web3 to communicate via Metamask, Wallet Connect or Portis.
```

**Review by cerberushades — CHANGES_REQUESTED** [review]

File: docs/develop/wallets/getting-started.md [hunk 5]
```diff
@@ -6,22 +6,84 @@ description: Build your next blockchain app on Polygon.
...
+[**CoinMarketCap**](https://coinmarketcap.com/currencies/polygon/).
+
+:::
+
+## Partner Wallets
+
+:::info *Non-native wallet support
+
+Wallets denoted with * in the table below are not natively supported with the wallet software 
+and require manual steps to add the Polygon network.
```
[the remark this answers is not in the document]
Comment (DannyS03) [reply docs/develop/wallets/getting-started.md]: Addressed

File: docs/develop/wallets/getting-started.md [hunk 6]
```diff
@@ -6,22 +6,84 @@ description: Build your next blockchain app on Polygon.
...
+
+:::
+
+:::info EOA: Externally Owned Account
+
+Externally owned accounts are controlled by a keypair and managed by the holder of the 
+private key, whereas smart contract accounts are managed by smart contract logic.
+
+:::
```
[the remark this answers is not in the document]
Comment (DannyS03) [reply docs/develop/wallets/getting-started.md]: made it into a general one

**Comment by DannyS03** [comment]
thanks @cerberushades, for the remaining suggestions, I did not update the rest of the content that the suggestions are made for as the plan is to change the content with a different approach when adding updates to the wallet content

**Comment by DannyS03** [comment]
came across something and going to make a few changes for how we approach the wallet content. I had made some additional updates to other pages but I think it's better to propose a new approach

File: docs/develop/wallets/getting-started.md [hunk 7]
```diff
@@ -6,45 +6,101 @@ description: Build your next blockchain app on Polygon.
...
   - docs
   - matic
-image: https://matic.network/banners/matic-network-16x9.png 
+image: https://matic.network/banners/matic-network-16x9.png
 ---
 
-The following article aims to be a guide through integration of a key management strategy on client side of your Decentralised Application on Matic Network.
+Wallets that support Polygon allow for key management, access to accounts controlled by 
+private keys, and interfaces that allow users to perform chain actions and sign transactions.
+The following page serves as a wallet index for wallets compatiable with Polygon. Please note
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:14]:
```suggestion
The following page serves as a wallet index for wallets compatible with Polygon. Please note
```

File: docs/develop/wallets/getting-started.md [hunk 8]
```diff
@@ -6,45 +6,101 @@ description: Build your next blockchain app on Polygon.
...
+:::
 
-* [**Metamask**](https://metamask.io/): Metamask is a browser add-on that manages a user’s Ethereum wallet by storing their private key on their browser’s data store and the seed phrase encrypted with their password. It is a non-custodial wallet, meaning, the user has full access and responsibility their private key. Once lost, the user can no longer control the savings or restore access to the wallet. [For more...](metamask)
+## Native Wallets
 
-* [**Portis**](https://www.portis.io/): Portis is a web-based wallet built keeping easy user-onboarding in mind. It comes with a javascript SDK that integrates into the DApp and creates a local wallet-less experience for the user. Further, it handles setting up the wallet, transactions and gas fees. [For more...](portis)
+[Polygon Support](https://support.polygon.technology/support/home) can support users and address
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:34]:
```suggestion
[Polygon Support](https://support.polygon.technology/support/home) can provide assistance to users and address 
```

Comment (cerberushades) [reply docs/develop/wallets/getting-started.md:34]: just avoiding the "support " repetition

File: docs/develop/wallets/getting-started.md [hunk 9]
```diff
@@ -6,45 +6,101 @@ description: Build your next blockchain app on Polygon.
...
+| [Trust Wallet](https://trustwallet.com/)* 	| non-custodial 	| EOA 	| no 	| support 	| yes 	|  	|  	| mobile 	|
+| [Unstoppable](https://unstoppable.money/) 	| non-custodial 	| EOA 	| no 	| no 	| yes 	|  	|  	| mobile, api/sdk 	|
+| [Venly](https://www.venly.io/) 	| hybrid 	| smart contract 	| no 	| interface 	| no 	|  	|  	| browser, api/sdk 	|
+| [Wirex](https://wirexapp.com/en/wirex-wallet)* 	| non-custodial 	| EOA 	| no 	| no 	| no 	|  	|  	| mobile 	|
+| [XDeFi](https://www.xdefi.io/) 	| non-custodial 	| EOA 	| no 	| interface 	| no 	| no 	| no 	| browser 	|
+| [Zerion](https://zerion.io/) 	| non-custodial 	| EOA 	| no 	| no 	| yes 	|  	|  	| mobile, browser 	|
+
+:::caution *Non-native wallet support
+
+Wallets denoted with * in the table below are not natively supported with the wallet software
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:90]:
```suggestion
Wallets denoted with * in the table above are not natively supported with the wallet software 
```

File: docs/develop/wallets/getting-started.md [hunk 10]
```diff
@@ -6,45 +6,101 @@ description: Build your next blockchain app on Polygon.
...
+| [PoS Wallet](https://wallet.polygon.technology/login/)           | non-custodial | EOA          | no        | no           | browser  |
+| [Hermez Wallet](https://wallet.hermez.io/login)                      | non-custodial | EOA          | no        | no           | browser  |
+| [Avail Apps](https://devnet-avail.polygon.technology/) (testnet) | non-custodial | EOA          | yes       | no           | browser  |
 
-* [**Wallet Connect**](https://walletconnect.org/): WalletConnect is an open protocol to communicate securely between Wallets and Dapps (Web3 Apps). The protocol establishes a remote connection between two apps and/or devices using a Bridge server to relay payloads. These payloads are symmetrically encrypted through a shared key between the two peers. [For more...](walletconnect)
 
+## Partner Wallets
 
-### High-level steps:
+The following wallets are solutions that Polygon Technology has partnered with.
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:46]:
```suggestion
The following wallets are solutions that Polygon Technology has partnered with:
```

File: docs/develop/wallets/getting-started.md [hunk 11]
```diff
@@ -6,45 +6,101 @@ description: Build your next blockchain app on Polygon.
...
+:::caution *Non-native wallet support
+
+Wallets denoted with * in the table below are not natively supported with the wallet software 
+and require manual steps to add the Polygon network.
+
+:::
+
+## Key Management Strategy
+
+The following basic steps allow for the integratation a client-side application with Polygon:
```
Comment (cerberushades) [inline docs/develop/wallets/getting-started.md:97]:
```suggestion
The following basic steps allow for the integration a client-side application with Polygon:
```

**Review by cerberushades — CHANGES_REQUESTED** [review]

**Review by cerberushades — APPROVED** [review]

**Comment by DannyS03** [comment]
@Nazeim @cerberushades PR was WIP as I was changing the content and I wanted to get the Wallet SME to check. Some fields aren't filled out for some wallets which need to be filled out

**Pull request merged**